From 6b557dc235257a9d4e69532f79007a9a4de854c6 Mon Sep 17 00:00:00 2001
From: David Hildenbrand <david@redhat.com>
Date: Tue, 17 Oct 2017 19:15:54 +0200
Subject: [PATCH 49/69] tools/kvm_stat: display message indicating lack of
 events

RH-Author: David Hildenbrand <david@redhat.com>
Message-id: <20171017191605.2378-29-david@redhat.com>
Patchwork-id: 77340
O-Subject: [RHEL-7.5 qemu-kvm-rhev PATCH 28/39] tools/kvm_stat: display message indicating lack of events
Bugzilla: 1497137
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

Upstream-status: linux.git 5725393764a342b6a5420fdd10184984ca08b5f6

commit 5725393764a342b6a5420fdd10184984ca08b5f6
Author: Stefan Raspl <raspl@linux.vnet.ibm.com>
Date:   Wed Jun 7 21:08:35 2017 +0200

    tools/kvm_stat: display message indicating lack of events

    Give users some indication on the reason why no data is displayed on the
    screen yet.

    Signed-off-by: Stefan Raspl <raspl@linux.vnet.ibm.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: David Hildenbrand <david@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 scripts/kvm/kvm_stat | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/kvm/kvm_stat b/scripts/kvm/kvm_stat
index 53dcd40..790fbce 100755
--- a/scripts/kvm/kvm_stat
+++ b/scripts/kvm/kvm_stat
@@ -1013,6 +1013,8 @@ class Tui(object):
                                    (key, values[0], values[0] * 100 / total,
                                     cur))
             row += 1
+        if row == 3:
+            self.screen.addstr(4, 1, 'No matching events reported yet')
         self.screen.refresh()
 
     def show_filter_selection(self):
-- 
1.8.3.1

